src/dialects/mariadb/index.js

```diff
@@ -24,6 +24,12 @@ export class MariaDbDialect extends AbstractDialect {
         ignoreDuplicates: ' IGNORE',
         updateOnDuplicate: ' ON DUPLICATE KEY UPDATE',
       },
+      addColumn: {
```

Comment:
It seems that MariaDB does actually support this; https://mariadb.com/kb/en/alter-table/

Reply:
bam bam this is awesome. @ephys how about I do it in subsequent PRs?

Reply:
I have a slight preference to do it in this PR directly, but I'm fine either way

Reply:
Same, I'm fine with either